app/models.py

```diff
@@ -76,7 +77,7 @@ def update_from_original(self, original):
                 current_app.logger.debug("{} has no column {} to copy from".format(original, c.name))
 
 
-class User(db.Model):
+class User(db.Model):  # type: ignore
```

Comment:
This file gave me the most trouble. `db.Model` is a type rather than a variable (or something like that?) so MyPy complains about using it as a base class, even though this seems to be the recommended way to use `flask-sqlalchemy`: https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/

I tried the suggested work around (https://github.com/dropbox/sqlalchemy-stubs/issues/76#issuecomment-595839159) but that didn't work. I tried ignoring the whole file, but that caused problems in other files because the db models no longer had type definitions. So I am resorting to adding comments that tell mypy to ignore individual lines.